modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -225,6 +225,109 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
 +----------------------------------------------------------------------------------------------------------------------------------------+
 ----
 
+[role=label--new-5.24]
+[[recreate-databases]]
+== Recreate databases using a procedure
+
+Neo4j 5.24 introduces the xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`] procedure, which allows you:
+
+* To restore a database store to a specified backup, while keeping all the users and roles (metadata) for the database.
+
+* To restore a database to a write mode again after it has been lost (for example, due to a disaster).
+See xref:clustering/disaster-recovery.adoc[] for more information.
+ 
+[CAUTION]
+====
+The recreate procedure works only for real user databases and not for composite databases, or the `system` database.
+
+Keep in mind the recreate procedure results in downtime while the stores get updated.
+The time is unbounded and may depend on the store size and network speed.
+
+The recreate procedure selects the most up-to-date seed from the defined seeding servers.
+If only a subset of the previous cluster members is specified as seeders, there is a risk of data loss.
+To avoid this, ensure you back up the current store before replacing it with a seed from a URI, so the latest state can be restored if needed.
```

Comment:
This is not really related to the sentence above. There is a risk of data loss when recreating the database if you choose you seeders "wrong" or the system database is not up to date. However, this sentence is more related to the first use case of recreate, when you want to go back in time.